test_tools/src/test_tools/odometer.py

```diff
@@ -96,7 +96,11 @@ def __init__(self, path=DEFAULT_PATH, filename=DEFAULT_FILENAME):
 
                 if found_header:
                     # Going over all lines.(this is the only option to get the last line) Doing nothing with other lines
-                    for last_row in reader:
+                    last_row = None
+                    try:
+                        for row in reader:
+                            last_row = row
```

Comment:
Unnecessary assignment inside the loop. Use `pass` and do the assignment outside the loop to make the code more readable.

Reply:
Please provide code suggestion as I don't get you completely.

Reply:
```python
try:
    for row in reader:
        pass
    last_row = row
except csv.Error:
    pass
```

This code is readable at one go.

Reply:
As discussed, this doesn't work. Also my code can't handle good lines after a bad line. WIP